commons/commons-bio/src/main/kotlin/ebi/ac/uk/api/Parameters.kt

```diff
@@ -1,5 +1,5 @@
 package ebi.ac.uk.api
 
 const val ON_BEHALF_PARAM = "onBehalf"
-const val REGISTER_PARAM = "sse"
+const val REGISTER_PARAM = "register"
```

Comment:
Good one. I always thought this name was kind of confusing. Could you please update the parameter description in the docs?

https://github.com/EBIBioStudies/biostudies-backend-services/tree/master/docs/api/submissions/operations